Formula/gofumpt.rb

```diff
@@ -0,0 +1,18 @@
+class Gofumpt < Formula
+  desc "Stricter gofmt"
+  homepage "https://github.com/mvdan/gofumpt"
+  url "https://github.com/mvdan/gofumpt/archive/v0.1.0.tar.gz"
+  sha256 "802c92d3df222c9b266d785305107c58a26ea186c4dbb5989b0db97b9bce0367"
+  license "BSD-3-Clause"
+
+  depends_on "go"
+
+  def install
+    system "go", "build", *std_go_args, "-ldflags", "-s -w -X main.version=#{version}", "-trimpath"
```

Comment:
This is already part of the std_go_args
```suggestion
    system "go", "build", *std_go_args, "-ldflags", "-s -w -X main.version=#{version}"
```

Reply:
done